.travis.yml

```diff
@@ -36,7 +36,7 @@ env:
 
 install:
   - git clone https://github.com/openresty/test-nginx.git test-nginx
-  - sudo luarocks install lua-resty-http
+  - sudo luarocks install lua-resty-http 0.14-0
```

Comment:
Why do we have to use the 14.0 version?

Reply:
I got a `unread buffer` exception from `lua-resty-http`, so I try to use last release version `0.14.0`,finally I found the `keep-alive` params to fix it.

Reply:
I got a `unread buffer` exception from `lua-resty-http` in etcd v3.4.0, so I try to use last release version `0.14.0`,finally I found the `keep-alive` params to fix it.

Reply:
https://github.com/iresty/lua-resty-etcd/blob/master/rockspec/lua-resty-etcd-master-0.1-0.rockspec#L16

I think we should keep same with this.

Reply:
https://github.com/iresty/lua-resty-etcd/pull/41/commits/3f0dc6399856b0a7fa0c7ded2353f08638995c5a

modify back to default